warmrestart/warmRestartAssist.h

```diff
@@ -108,7 +108,7 @@ class AppRestartAssist
     std::string m_appTableName;       // application table name
 
     bool m_warmStartInProgress;       // indicate if warm start is in progress
-    uint32_t m_reconcileTimer;        // reconcile timer value
+    time_t m_reconcileTimer;          // reconcile timer value
```

Comment:
please remove the trailing spaces

Reply:
If I remove ";_" trailing space after semicolon, the comment "//" will not start in same column as previous line.

Reply:
ah my bad 😭